tests/nn/moe/test_top2gating.py

```diff
@@ -27,17 +27,17 @@ def do_test_forward(device):
     gate = Top2Gate(4, 6).to(device)
     capacity = 2 * 12 // 6
     l_aux, combine_weights, dispatch_mask = gate(input)
-    assert pytest.approx(l_aux.item(), 0.0283)
+    assert pytest.approx(l_aux.item(), rel=0.01) == 0.0267, l_aux
```

Comment:
the old code seems to use pytest.approx incorrectly and new version detects this now. The numerical value that was used was in fact incorrect. Hence the change of the value.